### ISSUE TITLE
Sync process was interrupted

### ISSUE THREAD

**Issue by didli** [floccusaddon/floccus#1146]
### Which version of floccus are you using?
 Version 4.13.1

### Sync method
Nextcloud Bookmarks

### Which browser are you using?
Firefox 99.0.1 (Linux)

### Which version of Nextcloud? (if relevant)
Latest stable (23.0.4)

### Describe the Bug

I have setup floccus to send bookmarks (upload & overwrite cloud) to my nextcloud server once a day. It worked fine until Firefox 98. Now it will randomly give this message :
` E027: Sync process was interrupted ` 
Restarting Firefox then restart the sync will *sometimes* make it work. But it's far to be ideal, nor it is a sure thing.
Any idea please for what I should try to get my sync working again as it should ?
Log file :
[floccus-4.13.1-2022-04-28.log](https://github.com/floccusaddon/floccus/files/8586951/floccus-4.13.1-2022-04-28.log)
 

### Expected Behavior
A good sync once a day keeps the worries away.

### To Reproduce
No idea. It happens everyday, then suddenly worked for 2 days, then randomly stopped working again.

### Debug log provided
- [X] I have provided a debug log file

**Comment by marcelklehr** [comment]
Ahoy :)

The logs don't show anything suspicious. Lot's of folder reorderings and two moves. It could be that it's simply taking a long time and you are closing your browser or suspending your computer mid-sync. Is this plausible?

**Comment by didli** [comment]
Unfortunately, no. I don't use suspend (I don't even use a screensaver) and firefox is (almost) always up. At least, fireffox is UP when I'm desperately trying to sync again with my nextcloud server ^^. Do you think it could be related to another extension, one that can suspend tabs for example ?

**Comment by marcelklehr** [comment]
Another culprit could be the floccus version, which may have been updated in the meantime. Would you mind trying an older version? (e.g. v4.13 or v4.12)

**Comment by didli** [comment]
I'd prefer not to. I had CPU usage issues with the previous versions, which seems to be greatly mitigated (thank you !) with the version I'm running now. For now, I've found out that if I disable the autosync option and retry to sync, I can get a successfull sync faster (I've also tried to autosync with a short interval but it didn't work). Since it seems I'm the only one with this issue, I will close it. It should be most likely due to a specific configuration on my side. 
Thank you a lot for your help ^^!
